gulp/tasks/scripts.js

```diff
@@ -252,3 +252,9 @@ gulp.task( 'scripts',
     'scripts:ondemand'
   )
 );
+
+gulp.task( 'scripts:watch', function() {
+  gulp.watch( configScripts.src, gulp.parallel(
+    'scripts:modern'
+  ) );
```

Comment:
While this passes the linter, I think we would prefer the arguments to `gulp.watch` start on their own lines when one is a function.

```suggestion
  gulp.watch(
    configScripts.src,
    gulp.parallel( 'scripts:modern' )
  );
```